src/frontend/App/App.js

```diff
@@ -3,20 +3,43 @@ import { ToastContainer } from 'react-toastify';
 import SideBar from '../shared/SideBar/SideBar';
 import Header from '../shared/NavBar/Header';
 import MockAPI from '../mock/MockAPI';
-import { Login, Page404, SignUp, Home } from '../pages';
+import {
+	Login,
+	Page404,
+	SignUp,
+	Home,
+	History,
+	Likes,
+	Playlist,
+	WatchLater
+} from '../pages';
 import './App.css';
+import { useAuth } from '../context/auth-context';
 
 function App() {
+	const { user } = useAuth();
 	return (
 		<div className="app">
 			<Header />
 			<SideBar />
 			<div className="main-container">
 				<Routes>
 					<Route path="/" element={<Home />} />
-					<Route path="/login" element={<Login />} />
-					<Route path="/signup" element={<SignUp />} />
 					<Route path="/mockman" element={<MockAPI />} />
+					{!user.token && (
```

Comment:
Appreciation: It's good that you have given condition to routing !!

Reply:
Thank You